tests/MongoDB.Driver.Tests/MongoIndexManagerTests.cs

```diff
@@ -53,9 +64,9 @@ public class MongoIndexManagerTests
                 indexes.Count.Should().Be(2);
                 AssertIndex(collection.CollectionNamespace, indexes[0], "_id_");
                 var indexName = IndexNameHelper.GetIndexName(indexKeyDocument);
-                AssertIndex(collection.CollectionNamespace, indexes[1], indexName, expectedUnique: unique);
+                AssertIndex(collection.CollectionNamespace, indexes[1], indexName, expectedUnique: unique, expectedHidden: hidden);
 
-                void AssertIndex(CollectionNamespace collectionNamespace, BsonDocument index, string expectedName, bool expectedUnique = false)
+                void AssertIndex(CollectionNamespace collectionNamespace, BsonDocument index, string expectedName, bool expectedUnique = false, bool expectedHidden = false)
```

Comment:
Should local function be moved to the end of the enclosing function?

Reply:
done